packages/wdio-cli/src/commands/config.ts

```diff
@@ -158,22 +159,53 @@ export async function handler(argv: ConfigCommandArguments, runConfigCmd = runCo
     }
 }
 
+/**
+ * Helper utility used in `run` and `install` command to format a provided config path,
+ * giving it back as an absolute path, and a version without the file extension
+ * @param config the initially given file path to the WDIO config file
+ */
+export async function formatConfigFilePaths(config: string) {
+    const fullPath = config.includes(process.cwd())
+        ? config
+        : path.join(process.cwd(), config)
+
+    const fullPathNoExtension = fullPath.substring(0, fullPath.lastIndexOf('.'))
+
+    return { fullPath, fullPathNoExtension }
+}
+
+/**
+ * Helper utility used in `run` and `install` command to check whether a config file currently exists
+ * @param configPath the file path to the WDIO config file
+ */
+export async function canAccessConfigPath(configPath: string) {
+    return await fs.access(`${configPath}.js`).then(
+        () => true,
+        () => fs.access(`${configPath}.ts`).then(
+            () => true, () => false
+        )
+    )
+}
+
 /**
  * Helper utility used in `run` and `install` command to create config if none exist
  * @param {string}   command        to be executed by user
- * @param {string}   message        to show when no config is suppose to be created
+ * @param {string}   configPath     the path to a wdio.conf.[js/ts] file
  * @param {boolean}  useYarn        parameter set to true if yarn is used
  * @param {Function} runConfigCmd   runConfig method to be replaceable for unit testing
  */
-export async function missingConfigurationPrompt(command: string, message: string, useYarn = false, runConfigCmd = runConfigCommand) {
-    const configMessage = command === 'run'
-        ? `Error: Could not execute "run" due to missing configuration, file "${message}" not found! Would you like to create one?`
-        : `Error: Could not execute "${command}" due to missing configuration. Would you like to create one?`
+export async function missingConfigurationPrompt(command: string, configPath: string, useYarn = false, runConfigCmd = runConfigCommand) {
+
+    const message = (
+        `Could not execute "${command}" due to missing configuration, file` +
```

Comment:
Oops I missed a space here, will correct after we address all other outstanding issues.